src/xturing/engines/llama_engine.py

```diff
@@ -12,9 +15,9 @@ class LLamaEngine(CausalEngine):
     config_name: str = "llama_engine"
 
     def __init__(self, weights_path: Optional[Union[str, Path]] = None):
-        model_name = "sallywww/Llama-7B"
-        model = LlamaForCausalLM.from_pretrained(model_name)
-        tokenizer = LlamaTokenizer.from_pretrained(model_name)
+        model_name = "aleksickx/llama-7b-hf"
```

Comment:
Why are we changing the model here?

Reply:
Because to load model on colab, we need more sharded parts from the checkpoint. Otherwise it will crash because out of RAM.